test/apps/test_deeplabv3p_ar_detect_app.sh

```diff
@@ -2,32 +2,33 @@
 
 if [[ $# < 2 ]]
 then
-    echo "usage: test_deeplabv3p_ar_detect_app.sh [app prefix] " \
-         "[data root] [mpi exec] [test cores]"
+    echo "usage: test_deeplabv3p_ar_detect_app.sh [app prefix] "  \
+         "[data root] [assets root] [mpi exec] [test cores]"
```

Comment:
we should no longer have to pass this on the command line , use the functions you added in teca_py_config.i

Reply:
I applied this change only to the `test_deeplabv3p_ar_detect.py` test in [8a8bdb](https://github.com/LBL-EESA/TECA/pull/351/commits/8a8bdb2e85a86c212e899c369ed75a7750bae1a6). However I didn't do this to the app because the user is expected to pass the deeplab model path to the command line `--pytorch_deeplab_model`. That's why `[assets root]` is being passed to this bash file

Reply:
You need to update the command line application. You could leave the command line option, so that one could pass the model there. The default should be that the model is taken from the assets directory. This simplifies the use of the app. Having the model available internally(in the install) is the point of this work. There's no reason not to take advantage of it to make the tools more robust and easier to use.